packages/toast/src/toast.css

```diff
@@ -34,3 +34,27 @@ governing permissions and limitations under the License.
     transition-delay: 0s;
     visibility: visible;
 }
+
+:host([variant='error']),
```

Comment:
css added in to cover edge cases that werent known until storybook controls were added

Reply:
This should be driven from css

Reply:
i met with CSS and this is a temporary fix until the deprecated variants are removed.

Reply:
Can you please add a TODO in the code to make sure we follow up when the fix is in!